projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -1297,6 +1297,8 @@ export class IgxGridForOfDirective<T> extends IgxForOfDirective<T> implements On
             size = parseInt(this.igxForItemSize, 10) || 0;
             if (item && item.summaries) {
                 size = item.max;
+            } else if (item && item.groups && item.height) {
+                size = item.height;
```

Comment:
this requires that the height is assigned to the ```IGroupByRecord``` by the ```groupDataRecursive<T>``` method, which is not the case in 7.2.x. You should get that code as well from the 7.3.x PR

Reply:
@ChronosSF I updated the groupDataRecursive<T> to get the height as in the other PR

Reply:
In general for this related issue only the fix on line 670 would be enough.
The rest is for the group rows sizing when the regular rows have different height and so far it only happens for Multi Row Layout and it is not related to the IgxTreeGrid. Unless you want to bring the whole fix to 7.2.x as well.